errors/now-next-legacy-routes-optimized-lambdas.md

```diff
@@ -0,0 +1,16 @@
+# `@now/next` Legacy Routes Optimized Lambdas Opt-out
```

Comment:
```suggestion
# `@vercel/next` Legacy Routes Optimized Lambdas Opt-out
```

The file also should be renamed. I'd vote for dropping the prefix completely (`errors/next-legacy-routes-optimized-lambdas.md`)

Reply:
Updated